include/RAJA/config.hpp.in

```diff
@@ -395,6 +395,9 @@ const int DATA_ALIGN = @RAJA_DATA_ALIGN@;
 //
 #define RAJA_FORCEINLINE_RECURSIVE
 #define RAJA_INLINE inline  __attribute__((always_inline))
+//#define RAJA_UNROLL __attribute__((optimize("unroll-loops")))
```

Comment:
We should probably have an issue to circle back on the comments here and below. Please add one so we don't forget.

Reply:
actually, that should just get eliminated, I'll do that now.